apps/backend/core/configuration/validation/orchestrator.py

```
@@ -0,0 +1,450 @@
+        # Warning validators (log but continue)
+        self.register_validator(PusherValidator(self.config))
+        self.register_validator(ClerkValidator(self.config))
+        self.register_validator(SupabaseValidator(self.config))
+        self.register_validator(RobloxValidator(self.config))
+
+        # Info validators (optional)
+        self.register_validator(SentryValidator(self.config))
+
+    async def validate_all(
+        self,
+        show_table: bool = True,
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)